### ISSUE TITLE
Add ability to add aria-label and ref to Preview component.

### ISSUE THREAD

**Issue by adamfsage** [Sage/carbon#7967]
### Description

I am working on an accessibility ticket where loading states using Preview components must be communicated to users that are using a screenreader. Current preview implementation fails WCAG 4.1.3 Status Messages (AA).

As we don't use aria labels on Box components etc, we need to therefore be able to treat Preview components as a form of content. Whilst there is no text in Preview components, we are still communicating to the user that the component is loading through it's design. 

### Suggested solution

I want the Preview component to have `aria-label` and `ref` added as optional props.

My intention is for this is that once a loading state is triggered **through a user interaction**, the developer can focus the desired Preview component, where the aria-label on it will read out a loading message.

However, this should not be default behaviour for preview components. We don't want focus shifts every time we show Previews (eg. When we refresh the page and have multiple skeleton loaders for different areas). We only want this when the user explicitly is clicking a button to trigger a reload.

### Confidentiality

- [x] I confirm there is no confidential or commercially sensitive information included.

**Comment by nuria1110** [comment]
FE-7684

**Comment by adamfsage** [comment]
SBS-165101

**Comment by nicktitchmarsh** [comment]
The suggested solution to this is to use an aria-live region and populate that with updates or use a Loader instead of preview if screen reader updates are needed
